scripts-bootstrap.js

```diff
@@ -0,0 +1,24 @@
+require('shelljs/global')
+
+mkdir('-p', 'typings')
+
+touch('typings/main.d.ts')
+
+echo('SKIP_PREPUBLISH=true')
+env['SKIP_PREPUBLISH'] = true
+
+exec('npm install')
+exec('rimraf dist')
+
+if (!which('tsc')) {
```

Comment:
I'm not too sure; I've never used it before.  I based my approach on the npm docs [here](https://www.npmjs.com/package/shelljs).